nailgun/entities.py

```diff
@@ -2000,6 +2014,53 @@ def update_payload(self, fields=None):
         return {u'hostgroup': super(HostGroup, self).update_payload(fields)}
 
 
+class HostPackage(Entity):
+    """A representation of a Host Package entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        if _get_version(server_config) < Version('6.2'):
+            raise NotImplementedError(
```

Comment:
I saw this and it makes me wonder if would be good to add a deprecation warning when the user access the system entity on Satellite >= 6.2?